README.md

```diff
@@ -1,16 +1,230 @@
-# Title
+# JavaScript Fundamentals: Logical Operators
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+* Use `!` to negate an expression
+* Convert an expression to a boolean using `!!`
+* Link conditions with the `&&` and `||` operators
```

Comment:
This is just a suggestion, but when I struggle with SWBAT language, I usually fall back to something like "Identify how to use ...." Identify is a good learning goal word and it adapts to a lot of situations like this.